### PR TITLE
Release 4.0.0

### LINKED ISSUE #172: Puppet 7 support
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 7
- Ruby:-
- Distribution:-
- Module version:-

## What are you seeing
There is no support for puppet 7.
## What behaviour did you expect instead
Puppet 7 as supported version.
## Any additional information you'd like to impart
Hey,

has anyone already plans to update this module to support puppet7?

### PR DESCRIPTION
**Pull request by bk-lg**
#### Pull Request (PR) description
In refrerence to issue #172 I updated metadata.json and the corresponding documentation for a 4.0.0 release. 
I hope that a new major release os correct in your interpretation of semantic versioning.

#### This Pull Request (PR) fixes the following issues

Fixes #172

### CODE REVIEW

**Comment by bk-lg** [comment]
@smortex  The pipeline succeded but there are warnings about centos 8 with a timeout for the systemd service. I have at the moment no access to a centos vm for further investigation. Will this block a new release?

**Review by smortex — APPROVED** [review]
LGTM. The warning are due to pending tests (tests we know fail) so can be ignored.

I would like to have another approval form somebody at voxpupuli before merging and releasing a new version, thanks!

**Review by kenyon — APPROVED** [review]

**Comment by smortex** [comment]
> Shouldn't #172 be in the changelog?

I think only PR appear in the changelog and as noted in your other command, #164 includes it.

Let me merge and release this. Thanks @bk-lg !

**Pull request merged**
